test/sanity/globalConstants.wtest

```diff
@@ -0,0 +1,103 @@
+const x = 1 + 1
+test "global constants are referenceable" {
+  assert.equals(2, x)
+}
+
+
+const a = b
+const b = 1 + 1
+const c = b
+test "global constants referencing other constants defined before/after are properly initialized" {
+  assert.equals(2, a)
+  assert.equals(2, b)
+  assert.equals(2, c)
+}
+
+const o = p
```

Comment:
This reference const => wko is slightly more interesting than const => const, because of trivial spatial considerations, putting o after p changes readability. But still not crucial.